src/js/processor.js

```diff
@@ -0,0 +1,285 @@
+const constants = require('.\\constants.js');
+
+import info from '.\\infoBlock.js';
+
+// create html element with given attributes
+export function createHtml(selector, className, innerText, ...keysValues) {
+  const elem = document.createElement(selector);
+  className === undefined ? elem.className = '' : elem.className = className;
+  innerText === undefined ? elem.innerHTML = '' : elem.innerHTML = innerText;
+  if (keysValues.length !== 0) {
```

Comment:
what will be if I replace `keysValues.length !== 0` with `keyValues.length`?

pls repeat following article. https://learn.javascript.ru/type-conversions

Reply:
Done